docs/system_overview/data_objects.md

```diff
@@ -45,6 +45,49 @@ $ ils -l /tempZone/home/rods/foo
   rods              1 resc_1          284 2022-07-20.17:31 ? foo
 ```
 
+### Access Time
+
+iRODS tracks **access times for all replicas** to help administrators and users better understand data access patterns. This feature is designed for **performance and efficiency**, using a background queue mechanism.
+
+---
+
+#### When are access times updated?
+
+- Access time updates are triggered **when a replica is opened for reading**
+- An update is queued **only if**:
+    - The replica’s **last access time is earlier than its modification time**, or
+    - The **last access time is older than a configured threshold**
+
+Access time updates are processed **in the background** using a **shared memory message queue**. Queued updates **do not persist across server restarts**.
+
+!!! Tip
+    You can use [GenQuery](../system_overview/genquery.md) to retrieve replica access times and build custom reports or queries.
+
+---
+
+#### Configuration Options
```

Comment:
Now wondering whether something like this lives in a grid configuration section of the Configuration page. It's fine here for the time being.

Reply:
I've been wondering the same thing.

I think we need to include grid configuration options on the configuration page. Even if it's just to link to other parts of the site. Just makes it easier to find the info IMO.

Reply:
agreed.   especially now since the separation of client and server will be clearer / defensible.

Reply:
Created #337 to address that.